operator/apis/lifecycle/v1alpha1/keptnevaluationprovider_conversion.go

```diff
@@ -0,0 +1,56 @@
+package v1alpha1
+
+import (
+	"errors"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/keptn/lifecycle-toolkit/operator/api/v1alpha2"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+var ErrCastTo = errors.New("cannot be cast KeptnEvaluationProvider to v1alpha2")
+var ErrCastFrom = errors.New("cannot be cast KeptnEvaluationProvider to v1alpha1")
```

Comment:
please move it to common, like here https://github.com/keptn/lifecycle-toolkit/pull/569/files#diff-0b1df1465e8f9154d622e15002dada3cd33f305e84a7197fd3d7f2d2ff780f7bR36